packages/ember-testing/lib/events.js

```diff
@@ -1,25 +1,32 @@
-import { jQuery } from 'ember-views';
 import { run } from 'ember-metal';
+import { assign } from 'ember-utils';
 
 const DEFAULT_EVENT_OPTIONS = { canBubble: true, cancelable: true };
 const KEYBOARD_EVENT_TYPES = ['keydown', 'keypress', 'keyup'];
 const MOUSE_EVENT_TYPES = ['click', 'mousedown', 'mouseup', 'dblclick', 'mouseenter', 'mouseleave', 'mousemove', 'mouseout', 'mouseover'];
+const FORM_CONTROL_TAGS = ['INPUT', 'BUTTON', 'SELECT', 'TEXTAREA'];
 
 export function focus(el) {
   if (!el) { return; }
-  let $el = jQuery(el);
-  if ($el.is(':input, [contenteditable=true]')) {
-    let type = $el.prop('type');
+  if (el.isContentEditable || FORM_CONTROL_TAGS.indexOf(el.tagName) > -1) {
+    let type = el.getAttribute('type');
     if (type !== 'checkbox' && type !== 'radio' && type !== 'hidden') {
       run(null, function() {
+        let browserIsNotFocused = document.hasFocus && !document.hasFocus();
+
         // Firefox does not trigger the `focusin` event if the window
         // does not have focus. If the document doesn't have focus just
         // use trigger('focusin') instead.
+        if (browserIsNotFocused) {
+          fireEvent(el, 'focusin');
```

Comment:
FWIW, I totally realize that this isn't a mistake in your migrate to native from jQuery, and that it was preexisting.  During my testing of this functionality while migrating it to ember-test-helpers I don't think this is quite right, it should be something akin to this:

```js
  // makes `document.activeElement` be `element`. If the browser is focused, it also fires a focus event
  element.focus();

  // Firefox does not trigger the `focusin` event if the window
  // does not have focus. If the document does not have focus then
  // fire `focusin` event as well.
  if (browserIsNotFocused) {
    // if the browser is not focused the previous `el.focus()` didn't fire an event, so we simulate it
    fireEvent(element, 'focus', {
      bubbles: false,
    });

    fireEvent(element, 'focusin');
  }
```

Otherwise the event ordering is incorrect...
  

See [related implementation](https://github.com/emberjs/ember-test-helpers/blob/9836a34b782024e62e76321750f41bf2da44ee5d/addon-test-support/%40ember/test-helpers/dom/focus.js#L12-L27) from `@ember/test-helpers`.

Reply:
Getting these ie11 errors after refactoring to this. 

https://travis-ci.org/emberjs/ember.js/jobs/326476166

Looks like it is triggering the events multiple times and causing pain `click` triggering events in order.